### DISCUSSION TITLE
Unlimited lists

### DISCUSSION THREAD

**Discussion by k8scat** [community/community#9513] [Other Feature Feedback, Questions, & Ideas]
As a fullstack developer, 16 lists is not enough for me, would it be unlimited?

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/8633

**Reply by Cheney0077** [reply]
Allright

**Comment by cliffordfajardo** [comment]
I also hit the 16 limit limitation quickly. 
my use case:
- I typically view my GitHub newsfeed feed and GitHub trending list several times a week and I like to categorize repos i want to experiment with

Before GitHub lists, I created my own lists using an external store (notion database) but the need to context switch out of the GitHub web app  to save the repo and categorize it  usually caused me to just defer the action 😢 due to the extra step / time needed specially since I use GitHub on mobile a lot (I use it to catch up on updates / view GitHub feed like I do my twitter feed )

Reach 16 fast, 100 would keep us happy for a while 😄 
<img src="https://user-images.githubusercontent.com/6743796/160181648-ccda2b18-1423-4163-9126-1be922f3972b.png" height="300"/>


Related
- https://github.com/github/feedback/discussions/8608
- https://github.com/github/feedback/discussions/8633
- https://github.com/github/feedback/discussions/9672#discussion-3781923

**Reply by Kareka123** [reply]
اعتدال

**Comment by bougrina123** [comment]
hello
````

**Comment by Franky1** [comment]
I also hit a limit of 32 lists, that is too few for me... 
I like to structure everything ;-)
Can we have more lists?
Let's say 100 per user?

**Comment by sunxfancy** [comment]
I quickly hit a limit of 32 Lists. Since you can star thousands of projects, why not get hundreds of lists?
